packages/kernel-browser-runtime/CHANGELOG.md

```diff
@@ -7,11 +7,24 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.2.0]
+
+### Added
+
+- Add kernel command 'revoke' ([#544](https://github.com/MetaMask/ocap-kernel/pull/544))
+- Support multiple subclusters ([#530](https://github.com/MetaMask/ocap-kernel/pull/530))
+
+### Changed
+
+- Bump endo dependencies ([#543](https://github.com/MetaMask/ocap-kernel/pull/543))
```

Comment:
We should skip dependency updates unless they cause downstream changes, and if so we should express the changelog entry in terms of those changes.
```suggestion
```